theta/test/theta_sketch_test.cpp

```diff
@@ -296,4 +398,234 @@ TEST_CASE("theta sketch: conversion constructor and wrapped compact", "[theta_sk
   REQUIRE_THROWS_AS(wrapped_compact_theta_sketch::wrap(bytes.data(), bytes.size(), 0), std::invalid_argument);
 }
 
+TEST_CASE("theta sketch: wrap compact empty from java", "[theta_sketch]") {
+  std::ifstream is;
+  is.exceptions(std::ios::failbit | std::ios::badbit);
+  is.open(inputPath + "theta_compact_empty_from_java.sk", std::ios::binary | std::ios::ate);
+
+  std::vector<uint8_t> buf;
+  if(is) {
+      auto size = is.tellg();
```

Comment:
I would think that is.seekg(0, is.end) is needed first to obtain the file size with is.tellg()

Reply:
Nope, seekg isn't needed, because the file was opened with `std:iso:ate`, which seeks automatically to the end of the file when it is opened. Since we start at the end, all we need is `tellg` to determine where we are, and then a `seekg` to get back to the beginning.

Reply:
Right, I missed the "ate" flag. No problem then.